localization/ekf_localizer/src/ekf_module.cpp

```diff
@@ -38,6 +38,7 @@
 EKFModule::EKFModule(std::shared_ptr<Warning> warning, const HyperParameters params)
 : warning_(std::move(warning)),
   dim_x_(6),  // x, y, yaw, yaw_bias, vx, wz
+  accumulated_delay_times_(params.extend_state_step, 1.0E15),
```

Comment:
Is there a specific reason for initializing it with a very small value rather than 0?

Reply:
make sense. OK